templates/pgb_config.j2

```diff
@@ -0,0 +1,31 @@
+[databases]
+{% for name, database in databases.items() %}
+{{ name }} = host={{ database.host }} dbname={{ database.dbname }} port={{ database.port }} auth_user={{ database.auth_user }}
+{% endfor %}
```

Comment:
We should be able to split out most (all?) of the moving parts and use `%include` instead to reduce the amount of rerenders. Created [DPE-3549](https://warthogs.atlassian.net/browse/DPE-3549) to track.